.theme/pages/demo/error-handling/trigger-error-page.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <ContentRenderer :value="index" />
+  </div>
+</template>
+<script setup>
+import { showError } from "#app";
+import { queryContent } from "#imports";
+
+let index;
+
+// Searching for a non-existent markdown
+try{
```

Comment:
`try{` -> `try {`

bu suslu parantez oncesine bosluk eslint'te yok sanirim, onu ekleyebilirsen super olur

Reply:
bunu da bulamadim, ama bir sey warn olmus, ve if'lerden sonra space gelmis alakasiz. 

duz parantez'den once space istemedim `if(`, susluden once parantez dedim `try {`.

genel olarak isin bitmedigini biliyorum, ama bu comment'e sapka yollamissin ama anlamadim neyi cozdugunu

Reply:
eslint te rule ları arattığım zaman bunun kuralı için keyword-spacing i buldum. bu kuralı açınca try dan sonraki '{' ile if ten sonraki '(' i aynı kuralı uyguluyor. bunları ayırmak için override opsiyonu koymuşlar son son commit ile onu  ekledim şimdi artık 'if(' şeklinde 'try {' şeklinde istiyor zorunlu olarak